hardhat.config.ts

```diff
@@ -0,0 +1,35 @@
+import { HardhatUserConfig } from "hardhat/config";
+import "@nomicfoundation/hardhat-toolbox";
+import "@nomiclabs/hardhat-ethers";
+
+// The next line is part of the sample project, you don't need it in your
+// project. It imports a Hardhat task definition, that can be used for
+// testing the frontend.
+import "./tasks/accounts.ts";
+
+const config: HardhatUserConfig = {
+  solidity: "0.8.17",
+  networks: {
+    hardhat: {
+      chainId: 31337, // We set 1337 to make interacting with MetaMask simpler
+      accounts: {
+        mnemonic: "test test test test test test test test test test test junk",
+        path: "m/44'/60'/0'/0",
+        initialIndex: 0,
```

Comment:
can lines 18-29 be removed?

Reply:
Done

Reply:
![image](https://user-images.githubusercontent.com/9112273/224714101-067cb734-d66b-4707-8817-cfea31c9d4d4.png)
Why is it still not removed?